packages/material-ui/src/Modal/ModalManager.test.js

```diff
@@ -113,15 +122,22 @@ describe('ModalManager', () => {
     it('should handle the scroll', () => {
       const modal = {};
       const paddingRightBefore = container1.style.paddingRight;
+      const paddingFixedRightBefore = fixedNode.style.paddingRight;
```

Comment:
Is this just refactoring or did the test actually break?

Reply:
Both.

Reply:
So this is a breaking change after all or only breaking in jsdom?

Reply:
I think that it's only breaking jsdom.